testdata/invalid_file.go

```diff
@@ -1,3 +1,3 @@
 package p
 
-var i := 1
```

Comment:
Please revert this change

Reply:
It was auto fmt by `gofmt -s -w .`

Reply:
@wangzitian0 you should not run `gofmt -s -w` on `testdata` directory, just revert the change.

Reply:
Done.